message_definitions/v1.0/common.xml

```diff
@@ -2218,6 +2218,47 @@
         <param index="6">Empty.</param>
         <param index="7">Empty.</param>
       </entry>
+      <!-- AHRS commands (32000 to 32099) -->
+      <entry value="33000" name="MAV_CMD_AHRS_START" hasLocation="false" isDestination="false">
+        <description>Send command "start data sending" to the AHRS</description>
```

Comment:
```suggestion
        <description>Send command "start data sending" to the AHRS</description>
```

would be nice to find some neutral term for this.  If we were to try to apply this to DCM or EKF, how would we term that operation?

Also... why would we want to do this?  The answer to that might inform a better phrase here.

Reply:
Start/stop measuring by AHRS?